src/process.ts

```diff
@@ -219,10 +219,7 @@ export function runLintPass(
     return disposable;
   }
   const eslint = eslintPath;
-  // remove file:/Volumes/Macintosh HD from uri
-  const cleanPath = path
-    ? "/" + decodeURI(path).split("/").slice(5).join("/")
-    : null;
+  const cleanPath = path?.replace('file://', '');
```

Comment:
It looks like this causes a build failure when building locally. It builds and appears to work properly when changed to:
`const cleanPath = path ? path.replace('file://', '') : '';`

Reply:
Thanks for trying to build it! I've reverted parts to return `null` again, this should build now.